src/pages/careers/_model/_locations/_locations.ts

```diff
@@ -522,6 +523,31 @@ export const georgetown: LocationsType = {
     has_map: true,
 }
 
+export const roadtown: LocationsType = {
+    name: 'roadtown',
+    job_location: 'British-Virgin-Islands',
+    continent: 'caribbean',
+    display_continent: 'Caribbean',
+    link: '/careers/locations/roadtown/',
+    display_name: 'Road Town',
+    display_thumbnail_name: 'Road Town',
+    display_title_name: 'Road Town',
+    img_alt: ' Deriv’s Cayman Islands office at Road Town ',
```

Comment:
let's change the `image_alt` per SEO suggestion we having Cayman and Road town both here which they are separate office 🙏